task2/Task3.java

```diff
@@ -0,0 +1,34 @@
+package task2;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class Task3 {
+    public static void main(String[] args) {
+        Random random = new Random();
+        int[] positiveArray = new int[7];
+        int[] negativeArray = new int[7];
+        int[] array = new int[7];
+        int positiveCount = 7;
+        int negativeCount = 7;
```

Comment:
You can merge this variables to one like "count"

Reply:
Done